### PR TITLE
Ported VisionOn command (originally ShooterAim)

### PR DESCRIPTION
**Pull request by amull23**

### CODE REVIEW

File: Offseason2022/src/main/java/frc/robot/commands/VisionOn.java [hunk 1]
```diff
@@ -3,27 +3,35 @@
 
 package frc.robot.commands;
 
+import javax.lang.model.util.ElementScanner6;
```
Comment (jmullins16) [inline Offseason2022/src/main/java/frc/robot/commands/VisionOn.java:6]: Where did this import come from? I don't see this class being used (ElementScanner6).

File: Offseason2022/src/main/java/frc/robot/commands/VisionOn.java [hunk 2]
```diff
@@ -3,27 +3,35 @@
...
-    // addRequirements(m_subsystem);
+    m_lightOn = lightOn;
+    setName("VisionOn");
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize( )
-  {}
+  {
```
Comment (jmullins16) [inline Offseason2022/src/main/java/frc/robot/commands/VisionOn.java:29]: This is a robotcontainer reference like this:
RobotContainer.getInstance( ).m_vision.methodInVisionSubsystemToCall();

**Review by jmullins16 — CHANGES_REQUESTED** [review]
A couple of small items to fix.

**Review by jmullins16 — APPROVED** [review]
Looks good now!

**Review by zaraansari17 — COMMENTED** [review]
looks good!

**Review by hpark-24 — APPROVED** [review]
Looks good!

**Review by jmullins16 — APPROVED** [review]
Looks good now.

**Pull request merged**